arlo_server/sample_sizes.py

```diff
@@ -0,0 +1,54 @@
+from flask import jsonify, request
+from collections import defaultdict
+import math
+from werkzeug.exceptions import BadRequest
+
+from arlo_server import app, db
+from arlo_server.models import Election, Contest
+from arlo_server.routes import require_audit_admin_for_organization
+from audits import bravo, sampler_contest
+
+
+@app.route("/election/<election_id>/sample-sizes", methods=["GET"])
+def get_sample_sizes(election_id: str):
+    election = Election.query.get_or_404(election_id)
+    require_audit_admin_for_organization(election.organization_id)
+
+    if not election.contests:
+        raise BadRequest("Cannot compute sample sizes until contests are set")
+    if not election.risk_limit:
+        raise BadRequest("Cannot compute sample sizes until risk limit is set")
+
+    # For now, we only support one targeted contest
+    contest = next(c for c in election.contests if c.is_targeted)
+
+    # Sum the audit results from previous rounds
+    results_by_choice = defaultdict(int)
+    for result in contest.results:
+        results_by_choice[result.contest_choice_id] += result.result
+    results_so_far = {contest.id: results_by_choice}
+
+    # Do the math!
+    sample_sizes = bravo.get_sample_size(
+        election.risk_limit / 100,
+        sampler_contest.from_db_contest(contest),
+        results_so_far,
+    )
```

Comment:
@umbernhard I noticed that `bravo.get_sample_size` takes in the `sample_results` object (which I named `results_so_far` here), and that this object in the old single-jurisdiction code tried to aggregate results for every contest (even though there was only ever one contest). I figured that `bravo.get_sample_size` only considers the one contest that its passed anyway, so there was no point in passing in results for other contests. Does that sound right?

If so, might be worth changing the interface to `get_sample_size` to just take in the results for that one contest instead of a mapping of contest ids to results.